build.sh

```diff
@@ -44,6 +45,22 @@ if [ ! -d "$TOOLS_DIR" ]; then
   mkdir "$TOOLS_DIR"
 fi
 
+###########################################################################
+# INSTALL .NET 5 SDK
+###########################################################################
+
+echo "Installing .NET 5 SDK..."
```

Comment:
Do we need to do this? Can't we remove it now that Azure Pipelines handles it for us?